src/e2e-test/java/io/cdap/plugin/pubsub/actions/PubSubActions.java

```diff
@@ -85,4 +89,9 @@ public static void enterSubscription(String subscription) {
   public static void enterNumberOfReaders(String numberOfReaders) {
     ElementHelper.replaceElementValue(PubSubLocators.numberOfReaders, numberOfReaders);
   }
+
+  public static void selectDataType() {
+    Select select = new Select(PubSubLocators.messageDataType);
+    select.selectByIndex(9);
```

Comment:
Why are we hardcoding the index here are we sure the desired message data type will be at 9th index at all occurrences ?

Reply:
yes.